src/components/ObsEdit/Sheets/NotesSheet.js

```diff
@@ -0,0 +1,41 @@
+// @flow
+
+import { TextInputSheet } from "components/SharedComponents";
+import type { Node } from "react";
+import React, { useCallback } from "react";
+import useTranslation from "sharedHooks/useTranslation";
+
+type Props = {
+  setShowNotesSheet: Function,
+  addNotes: Function,
+  description: ?string
+}
+
+const NotesSheet = ( {
+  setShowNotesSheet,
+  addNotes,
+  description
+}: Props ): Node => {
+  const { t } = useTranslation( );
+
+  const handleClose = useCallback(
+    ( ) => setShowNotesSheet( false ),
+    [setShowNotesSheet]
+  );
+
+  return (
+    <TextInputSheet
+      handleClose={setShowNotesSheet}
+      headerText={t( "NOTES" )}
+      snapPoints={[416]}
+      placeholder={t( "Add-optional-notes" )}
+      initialInput={description}
+      confirm={textInput => {
+        addNotes( textInput );
+        handleClose( );
+      }}
+    />
+  );
+};
+
+export default NotesSheet;
```

Comment:
Does this component need to exist? I feel like you've done such a good job abstracting things into the reusable `<TextInputSheet>` that you could just use that directly inside `<OtherDataSection>`... unless you think we're going to reuse `NotesSheet` with exactly the same configuration somewhere.